build.gradle

```diff
@@ -32,7 +32,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.3'
+        classpath 'com.android.tools.build:gradle:3.3.0-alpha02'
```

Comment:
I think if we use 3.3.0, we need to update gradle version too? I guess we might limit this PR to just purely update Kotlin version.